src/main/scala/org/opennetworkinsight/dns/DNSWordCreation.scala

```diff
@@ -3,17 +3,32 @@ package org.opennetworkinsight.dns
 import org.apache.spark.SparkContext
 import org.apache.spark.broadcast.Broadcast
 import org.apache.spark.rdd.RDD
-import org.apache.spark.sql.{DataFrame, Row, SQLContext}
 import org.apache.spark.sql.functions._
 import org.apache.spark.sql.types.{DoubleType, StringType, StructField, StructType}
-import org.opennetworkinsight.utilities.{Entropy, Quantiles}
+import org.apache.spark.sql.{DataFrame, Row, SQLContext}
 import org.opennetworkinsight.dns.{DNSSchema => Schema}
+import org.opennetworkinsight.utilities.{Entropy, Quantiles}
 import org.slf4j.Logger
 
 import scala.io.Source
 
   object DNSWordCreation {
 
+    val countryCodes = Set("ac", "ad", "ae", "af", "ag", "ai", "al", "am", "an", "ao", "aq", "ar", "as", "at", "au",
```

Comment:
I guess we can read from a file instead of this long variable. 
Will keep in mind for the future.